eslint-common.json

```diff
@@ -90,6 +90,10 @@
     "yoda": ["error", "never", {"exceptRange": true}]
   },
   "overrides": [
+    {
+      "files": ["*.mjs"],
+      "parserOptions": {"sourceType": "module"}
+    },
```

Comment:
I just tested it, and it seems this rule correctly merges with overlapping rules. :)

Reply:
Very nice!